package.json

```diff
@@ -9,17 +9,19 @@
   },
   "scripts": {
     "postinstall": "./scripts/postinstall.sh",
-    "test": "./node_modules/jest-cli/bin/jest.js",
-    "lint": "./node_modules/eslint/bin/eslint.js src/"
+    "coveralls": "./node_modules/mocha/bin/mocha --require blanket --require tests/bootstrap.js --reporter mocha-lcov-reporter --compilers .:tests/compiler.js src/**/__tests__/*-test.js src/components/**/__tests__/*-test.js | ./node_modules/coveralls/bin/coveralls.js",
+    "test": "./node_modules/mocha/bin/mocha --require tests/bootstrap.js --compilers .:tests/compiler.js src/**/__tests__/*-test.js src/components/**/__tests__/*-test.js",
+    "lint": "./node_modules/eslint/bin/eslint.js src/ && echo '🍻 All good!'"
```

Comment:
I've been told this in the past as well. Sadly, it doesn't work. All commands fail locally for me when I remove the path. I'm going to leave this as-is given it works and changing it in the future won't modify how the commands are ran by developers.